lib/entities/default_settings_migration.dart

```diff
@@ -259,6 +259,10 @@ Future<void> defaultSettingsMigration(
         case 42:
           updateBtcElectrumNodeToUseSSL(nodes, sharedPreferences);
           break;
+        case 43:
+          await changeDefaultMoneroNode(nodes, sharedPreferences);
+          break;
```

Comment:
how is calling this function going to make the node trusted?
like this it will only delete the node and add it back, which will still default `trusted` to false

make a new function that would just change the trusted boolean for our node to true